message/bdoor.go

```diff
@@ -0,0 +1,124 @@
+package message
+
+const (
+	VMWARE_BDOOR_MAGIC  = uint64(0x564D5868)
+	VMWARE_BDOOR_PORT   = uint16(0x5658)
+	VMWARE_BDOORHB_PORT = uint16(0x5659)
+
+	VMWARE_BDOOR_CMD_GETVERSION           = uint32(10)
+	VMWARE_BDOOR_CMD_GETDEVICELISTELEMENT = uint32(11)
+	VMWARE_BDOOR_CMD_TOGGLEDEVICE         = uint32(12)
+
+	VMWARE_BDOOR_CMD_MESSAGE   = uint16(0x1e)
+	VMWARE_BDOORHB_CMD_MESSAGE = uint16(0)
+	GUESTMSG_FLAG_COOKIE       = uint32(0x80000000)
+)
+
+type BackdoorProto struct {
```

Comment:
maybe the low-level backdoor accessors should be made private?
Or alternately they could be put in a separate `github.com/vmware/vmw-guestinfo/bdoor` package, but in any case I don't think such constructs should appear in the documentation for the `message` package